WordPress/wporg-main-2022
# Data Liberation: Add interior pages

This PR adds an interface for the guide location in the https://github.com/WordPress/data-liberation repository.

 
### Screenshots


| Header | Header |
|--------|--------|
| Home | <img src="https://github.com/WordPress/wporg-main-2022/assets/1657336/bdb1f194-c7d7-4003-9dc2-7908d9a8abd1"> |
| Archive | ![localhost_8888_data-liberation_and_ (1)](https://github.com/WordPress/wporg-main-2022/assets/1657336/ff22ec63-ca55-423e-a051-4ca2ff94f941) |
| Single | ![localhost_8888_data-liberation_and_rss-4_](https://github.com/WordPress/wporg-main-2022/assets/1657336/d7c8f9cd-bdbc-4908-9a92-5ee986a830ae) | 




### How to test the changes in this Pull Request:

- run `yarn wp-env run cli wp cron event run --due-now`
  - See this PR for background: https://github.com/WordPress/wporg-main-2022/pull/374 
- Create a page `data-liberation`
  - Associate the `page-data-liberation` template
- Create a child page of `data-liberation` with the slug `and`
